README.md

```diff
@@ -102,6 +102,25 @@ from kymatio.tensorflow import Scattering2D
 scattering = Scattering2D(J=2, shape=(32, 32))
 ```
 
+## Keras
+
+After installing the latest version of `keras`, you can call `Scattering2D` as a `tf.Module` via for instance:
```

Comment:
It's just TF. You don't install Keras separately anymore. Also `tf.Module` should be just “Keras `Layer`”.